### PR TITLE
Nit: Updated import statement with correct point

### LINKED ISSUE #756: add full import statement to `server` option example
I guess this [Integration Reference Page](https://docs.astro.build/en/reference/integrations-reference/#server-option) is missing full import statement.

### PR DESCRIPTION
**Pull request by aFuzzyBear**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=Rxqsvy8)

#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- Closes  #756 

Re-pointed the import to the correct location

### CODE REVIEW

File: src/pages/en/reference/integrations-reference.md [hunk 1]
```diff
@@ -194,15 +194,20 @@ A read-only copy of the user-supplied [Astro config](/en/reference/configuration
...
+import partytown from '@astrojs/partytown'
+
+export default {
+  name: 'partytown'
+  hooks:{
+    'astro:server:setup': ({ server }) => {
+      server.middlewares.use(
+        partytownServer(partytownLibDirectory, {
+          mount: '/~partytown',
+          ...
```
Comment (delucis) [inline src/pages/en/reference/integrations-reference.md:206]:
```suggestion
          // ...
```

File: src/pages/en/reference/integrations-reference.md [hunk 2]
```diff
@@ -194,15 +194,20 @@ A read-only copy of the user-supplied [Astro config](/en/reference/configuration
...
-    partytownServer(partytownLibDirectory, {
-      mount: '/~partytown',
-      ...
-    })
-  );
+import partytown from '@astrojs/partytown'
+
+export default {
+  name: 'partytown'
+  hooks:{
```
Comment (delucis) [inline src/pages/en/reference/integrations-reference.md:201]:
```suggestion
  hooks: {
```

File: src/pages/en/reference/integrations-reference.md [hunk 3]
```diff
@@ -194,15 +194,20 @@ A read-only copy of the user-supplied [Astro config](/en/reference/configuration
...
-
-'astro:server:setup': ({ server }) => {
-  server.middlewares.use(
-    partytownServer(partytownLibDirectory, {
-      mount: '/~partytown',
-      ...
-    })
-  );
+import partytown from '@astrojs/partytown'
+
```
Comment (delucis) [inline src/pages/en/reference/integrations-reference.md:198]: Changed my mind on the import sorry as we can’t actually import the middleware, it’s probably just misleading to show it.

```suggestion
```

File: src/pages/en/reference/integrations-reference.md [hunk 4]
```diff
@@ -194,15 +194,20 @@ A read-only copy of the user-supplied [Astro config](/en/reference/configuration
...
+export default {
+  name: 'partytown'
+  hooks: {
+    'astro:server:setup': ({ server }) => {
+      server.middlewares.use(
+        partytownServer(partytownLibDirectory, {
+          mount: '/~partytown',
+          // ...
+        })
```
Comment (delucis) [inline src/pages/en/reference/integrations-reference.md:207]: I think we can make this more generic. This suggestion is based on [the signature used inside the partytown integration](https://github.com/withastro/astro/blob/08432d5b01a5cab1bd8d962c30f77ab3827bc5aa/packages/integrations/partytown/src/sirv.ts#L205).

```suggestion
        function middleware(req, res, next) {
          // handle requests
        }
```

**Review by delucis — COMMENTED** [review]
Re-read this and realised we might be being misleading, so here are some suggestions.

**Review by delucis — APPROVED** [review]
Lovely stuff — LGTM when those check pass. Thanks @aFuzzyBear 🙌

**Review by Pukimaa — APPROVED** [review]
LGTM too!

**Comment by sarah11918** [comment]
Update branch and merge, @aFuzzyBear !  ❤️

**Pull request merged**